java/code/src/com/redhat/rhn/domain/kickstart/crypto/CryptoKey.java

```diff
@@ -135,17 +135,12 @@ public void setKey(byte[] keyIn) {
      * @return String version of the key.
      */
     public String getKeyString() {
-        try {
-            if (this.key != null) {
-                String retval = new String(this.key, "UTF-8");
-                if (!StringUtils.isEmpty(retval)) {
-                    return retval;
-                }
+        if (this.key != null) {
+            String retval = new String(this.key, StandardCharsets.UTF_8);
+            if (!StringUtils.isEmpty(retval)) {
+                return retval;
             }
```

Comment:
As you are touching this and also `BaseCryptoKeyCommand` where this is also consumed, can we skip returning `null` and rather check of empty string on consumer side?

I mean it is marked as String and true, it can be null but we have Optionals everywhere else and here we pass null as a String.

Reply:
makes sense. I think the other uses of this shouldn't have problems: most of them are tests